src/historical/HistoricalProviderSpec.js

```diff
@@ -1,124 +1,113 @@
+import HistoricalProvider from './HistoricalProvider';
 
-define([
-    './HistoricalProvider'
-], function (
-    HistoricalProvider
-) {
-    'use strict';
+describe('Historical Provider', function () {
+    let historicalProvider;
+    let mockOpenMct;
+    let mockMetadata;
+    let options;
+    let mockDomainObject;
+    let mockSessionService;
+    let mockNotification;
+    let resolveCallback;
+    let rejectCallback;
 
-    xdescribe('Historical Provider', function () {
-        let historicalProvider;
-        let mockOpenMct;
-        let mockMetadata;
-        let options;
-        let mockDomainObject;
-        let mockSessionService;
-        let mockNotification;
-        let resolveCallback;
-        let rejectCallback;
+    beforeEach(function () {
+        options = {
+            domain: 'scet'
+        };
+        mockOpenMct = {};
 
-        beforeEach(function () {
-            options = {
-                domain: 'scet'
-            };
-            mockOpenMct = {};
-
-            mockOpenMct.telemetry = jasmine.createSpyObj('telemetryApi', [
-                'getMetadata'
-            ]);
-            mockMetadata = jasmine.createSpyObj('metadata', [
-                'value',
-                'values'
-            ]);
-            mockMetadata.values.and.returnValue([]);
-            mockOpenMct.telemetry.getMetadata.and.returnValue(mockMetadata);
-            mockMetadata.value.and.returnValue({
-                'key': 'scet'
-            });
+        mockOpenMct.telemetry = jasmine.createSpyObj('telemetryApi', [
+            'getMetadata'
+        ]);
+        mockMetadata = jasmine.createSpyObj('metadata', [
+            'value',
+            'values'
+        ]);
+        mockMetadata.values.and.returnValue([]);
+        mockOpenMct.telemetry.getMetadata.and.returnValue(mockMetadata);
+        mockMetadata.value.and.returnValue({
+            'key': 'scet'
+        });
 
-            mockSessionService = jasmine.createSpyObj('sessionService', [
-                'getHistoricalSession'
-            ]);
-            mockOpenMct.$injector = jasmine.createSpyObj('injector', [
-                'get'
-            ]);
-            mockOpenMct.$injector.get.and.returnValue(mockSessionService);
-            mockOpenMct.notifications = jasmine.createSpyObj('notificationAPI', [
-                'alert'
-            ]);
-            mockNotification = jasmine.createSpyObj('notification', [
-                'on',
-                'off'
-            ]);
-            mockOpenMct.notifications.alert.and.returnValue(mockNotification);
+        mockSessionService = jasmine.createSpyObj('SessionService', [
+          'getHistoricalSession'
+        ]);
+        spyOn(HistoricalProvider.prototype, 'getSessionService').and.returnValue(mockSessionService);
 
+        mockOpenMct.notifications = jasmine.createSpyObj('notificationAPI', [
+            'alert'
+        ]);
+        mockNotification = jasmine.createSpyObj('notification', [
```

Comment:
what's this 'on' and 'off'? I don't think we have that on notifications (maybe we did in the past?)